docs/source/gbd2019_models/risk_exposures/bmi/index.rst

```diff
@@ -141,68 +141,60 @@ Restrictions
      - Value
      - Notes
    * - Male only
-     -
+     - False
      -
    * - Female only
+     - False
      -
+   * - YLD only
+     - False
      -
-   * - Age group start
-     -
+   * - YLL only
+     - False
      -
+   * - Age group start
+     - 9
+     - [20, 24 years)
    * - Age group end
-     -
-     -
-
-..	todo::
-
-	Determine if there's something analogous to "YLL/YLD only" for this section
+     - 235
+     - [95, 125 years)
 
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-Describe the clinical and mathematical assumptions made for this cause model,
-and the limitations these assumptions impose on the applicability of the
-model.
-
-Risk Exposure Model Diagram
-++++++++++++++++++++++
+The quantity of interest is exposure to the mean BMI level; we assume full reversibility of risk and do not account for duration of exposure to BMI values above the range of the TMREL. 
 
-Include diagram of Vivarium risk exposure model.
 
-Data Description Tables
-+++++++++++++++++++++++
+Data Description
+++++++++++++++++
 
-As of 02/10/2020: follow the template created by Ali for Iron Deficiency, copied 
-below. If we discover it's not general enough to accommodate all exposure types,
-we need to revise the format in coworking. 
+The rei_id for BMI is 108
 
-.. list-table:: Constants 
+.. list-table:: ID Table 
 	:widths: 10, 5, 15
 	:header-rows: 1
 
-	* - Constant
-	  - Value
-	  - Note
-	* - 
+	* - Component
+	  - ME_ID
+	  - Notes
+	* - Mean exposure
+	  - 2548
 	  - 
+	* - Standard deviation
+	  - 18706
 	  - 
+	* - Relative risk
+	  - 9031
+	  - Must be accessed with get_draws; adult values
```

Comment:
What relative risk(s) does this refer to? Is it for a specific risk-outcome pair or something else? I could use clarification here otherwise it looks good to me!